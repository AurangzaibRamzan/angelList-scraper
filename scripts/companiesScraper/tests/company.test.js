const companyScrapper = require('../company');


describe("scrape companies", () => {
  it("wonolo", async () => {
    const company = await companyScrapper('https://angel.co/company/wonolo')
    expect(company).toMatchSnapshot();
  });
  it("stord", async () => {
    const company = await companyScrapper('https://angel.co/company/stord')
    expect(company).toMatchSnapshot();
  });
});


