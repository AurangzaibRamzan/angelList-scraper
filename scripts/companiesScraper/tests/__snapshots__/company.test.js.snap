// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scrape companies stord 1`] = `
Object {
  "companySize": "51-200 people",
  "description": "The Supply Chain CloudStord is the supply chain in the cloud—empowering brands to build sophisticated, agile, and integrated supply chains. With a single integration between your enterprise systems to the Stord Supply Chain Cloud, you unlock a single view of inventory, order orchestration logic, audited data and analytics, and more. Backed by our network of 500+ warehouses, 30+ fulfillment centers and 20k+ carriers, use the Stord Supply Chain Cloud to request a shipment, launch a new warehouse or run new distribution models that help you out-deliver the competition. Let’s move supply chains to the cloud, together.  We're a rapidly growing team that is backed by over $15M from leading investors such as Kleiner Perkins, Susa Ventures, Dynamo, and more.  Learn more about STORD, our business, and career opportunities at stord.com",
  "links": Object {
    "facebook": "http://facebook.com/stordco",
    "linkedin": "https://www.linkedin.com/company/stord/",
    "twitter": "http://twitter.com/getstord",
    "website": "http://www.stord.com",
  },
  "locations": Array [
    "Atlanta",
  ],
  "marketTags": Array [
    "Information Technology",
    "Real Time",
    "Services",
    "Supply Chain Management",
    "Software",
    "Technology",
    "Logistics",
    "Warehouse And Chain Management",
    "Data Warehousing",
  ],
  "member": Array [
    Object {
      "name": "Jacob Boudreau",
      "tilte": "CTO",
    },
    Object {
      "name": "Sean Henry",
      "tilte": "Founder",
    },
  ],
  "name": "STORD",
  "raised": "$46.2M",
  "round": Object {
    "foundOver": "3 rounds",
    "round": "Series B (Nov 2020)",
  },
  "tag": "The Supply Chain Cloud",
  "url": "https://angel.co/company/stord",
}
`;

exports[`scrape companies wonolo 1`] = `
Object {
  "companySize": "51-200 people",
  "description": "We help underemployed workers find flexible work in minutes at businesses across the USWonolo, which stands for Work Now Locally, is disrupting the $70bn temporary staffing industry. Founded in 2014, Wonolo is a remote-first company with three regional hubs: San Francisco Bay Area, Nashville, and Toronto.   We have built a tech-enabled marketplace that connects, within minutes, businesses in need of front-line workers on a temporary or seasonal basis, with 300,000 underemployed workers in local markets all across the country. Our mission is to empower the in-demand workforce by providing flexible and fulfilling job opportunities for everyone.  We are backed by leading investors including Sequoia, Bain Capital, PivotNorth, and Base10. We’re passionate about solving tough problems with a technology platform that empowers people to choose work that works for them instead of the other way around.  Wonolo is a Great Place to Work Certified company for delivering a consistently high-trust experience for all employees. We value diversity and are committed to creating an inclusive work environment. We provide equal employment opportunity for all applicants and employees.  #23 in Fortune's Best Workplaces in Technology 2020 (Small and Medium) #16 in Fortune's Best Workplaces in the Bay Area 2020 (Small and Medium) #40 in Best Small and Medium Workplaces 2019",
  "links": Object {
    "facebook": "http://www.facebook.com/wonoloapp",
    "linkedin": "http://www.linkedin.com/company/wonolo",
    "twitter": "http://twitter.com/wonolo",
    "website": "http://wonolo.com",
  },
  "locations": Array [
    "San Francisco",
    "Toronto",
    "Nashville",
  ],
  "marketTags": Array [
    "Marketplaces",
    "Local",
    "Startups",
    "Temporary Staffing",
    "Staffing Firms",
    "Marketplace",
  ],
  "member": Array [
    Object {
      "name": "Yong Kim",
      "tilte": "Founder",
    },
    Object {
      "name": "AJ Brustein",
      "tilte": "Founder",
    },
  ],
  "name": "Wonolo",
  "raised": "$52.9M",
  "round": Object {
    "foundOver": "4 rounds",
    "round": "Series C (Nov 2018)",
  },
  "tag": "We help underemployed workers find flexible work in minutes at businesses across the US",
  "url": "https://angel.co/company/wonolo",
}
`;
